twitchio/websocket.py

```diff
@@ -483,5 +483,8 @@ async def _close(self):
         for fut in futures:
             fut.cancel()
 
-        await self._websocket.close()
+        if self._websocket:
+            await self._websocket.close()
+        if self._client._http.session:
+            await self._client._http.session.close()
         self._loop.stop()
```

Comment:
In correlation to your other PR (#248) I would point out that self._loop.stop is called in _close. I'd think that changing this to close, and removing the close call in your other pr would work to fix that one. Other than that, LGTM

Reply:
Thanks for the quick review and for the feedback.

Unrelated to this patch and this PR, but related to #248, I have some reservations about calling ``loop.stop()`` or ``loop.close()``, because this breaks other processes that are using the event loop. There might be some async cleanup tasks that an application may wish to await for after ``bot.close()``. One thing at a time though...

Reply:
I myself intend to use this package in the Home Assistant project. `loop.stop()` or `loop.close()` are both devastating, crashing the entire program.